workspaces/liferay-learn-workspace/client-extensions/liferay-learn-custom-element/src/components/LearningPath/LearningPathList.jsx

```diff
@@ -3,12 +3,46 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-import React from 'react'
+import "../../index.scss";
+import ClayLayout from "@clayui/layout";
+import { useEffect, useState } from "react";
+import { getLearningPaths } from "../../services/learning-path";
+import SectionCard from "../Common/SectionCard";
 
-const LearningPathList = () => {
-  return (
-    <div>LearningPathList</div>
-  )
-}
+const LearningPathsList = () => {
+    const [learningPaths, setLearningPaths] = useState(null);
 
-export default LearningPathList
\ No newline at end of file
+    useEffect(() => {
+        getLearningPaths(3)
+            .then(data => setLearningPaths(data))
+            .catch(error =>
+                console.error('Error fetching learning paths:', error)
+            );
+    }, []);
+
+    return (
+        <ClayLayout.ContainerFluid view>
+            <ClayLayout.Row justify="start">
+                {learningPaths && learningPaths.length > 0 && (
+                    <>
+                        {learningPaths.map((learningPath, index) => {
+                            return (
+                                <SectionCard
+                                    description={learningPath.description}
```

Comment:
Please, add the verification for description and title. 
Follow the course model.